rmw_dds_common/test/test_graph_cache.cpp

```diff
@@ -1391,6 +1382,9 @@ TEST(test_graph_cache, bad_arguments)
   }
 
   {
+    rcutils_allocator_t allocator = rcutils_get_default_allocator();
+    rcutils_string_array_t names = rcutils_get_zero_initialized_string_array();
+    rcutils_string_array_t namespaces = rcutils_get_zero_initialized_string_array();
```

Comment:
where are the strings array freed?

Reply:
Good catch! See https://github.com/ros2/rmw_dds_common/pull/22/commits/4d921b5db21e6b7bb6cdacbe05ba768daf92da84.